### ISSUE TITLE
ChannelMultiple raises exception if there are no outputs

### ISSUE THREAD

**Issue by comex** [ebb29c/asyncio-channel#1]
This example creates a ChannelMultiple, then writes to its input channel without adding any outputs:
```py
import asyncio_channel, asyncio
async def main():
    src = asyncio_channel.create_channel()
    mult = asyncio_channel.create_multiple(src)
    await src.put(42)
asyncio.get_event_loop().run_until_complete(main())
```

I would expect this to simply drop the message with no errors.  A ChannelMultiple with 2 outputs sends 2 messages; a ChannelMultiple with 1 output sends 1 message; so a ChannelMultiple with 0 outputs should send 0 messages.

Instead, I get:
```
Task exception was never retrieved
future: <Task finished name='Task-2' coro=<_start() done, defined at /Users/comex/src/asyncio-channel/asyncio_channel/_create_multiple.py:16> exception=ValueError('Set of coroutines/Futures is empty.')>
Traceback (most recent call last):
  File "/Users/comex/src/asyncio-channel/asyncio_channel/_create_multiple.py", line 22, in _start
    await wait_all(*(_put(ch, x) for ch, _ in outs))
  File "/Users/comex/src/asyncio-channel/asyncio_channel/_util.py", line 23, in wait_all
    done, pending = await wait(tasks, timeout=timeout)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/asyncio/tasks.py", line 413, in wait
    raise ValueError('Set of coroutines/Futures is empty.')
ValueError: Set of coroutines/Futures is empty.
```

**Comment by ebb29c** [comment]
Hello, thanks for reporting this.  Sorry that I did not see this until now.  Apparently my notification settings were off.

**Comment by ebb29c** [comment]
Fix has been released in v0.9.1
